lib/fabrik/DirectorManager.js

```diff
@@ -211,6 +211,11 @@ class DirectorManager extends BaseManager {
                 previous_manifest: manifest
               }, opts.context);
               return;
+            })
+            .then(() => this.executePreUpdateActions(deploymentName, actionContext))
+            .tap(response => {
+              logger.info('PreUpdate action response is ...', response);
```

Comment:
logger will become very difficult to decipher without context info of `deploymentName` in log statement. Please include deployment name in logger. 
```
logger.info(`PreUpdate action response is for ${deploymentName}...`, response);
```